README.md

```diff
@@ -837,6 +838,22 @@ To get best results from HA mode some additional configurations might require:
 The metrics are served through https://localhost:10258/metrics by default.
 The address and port can be changed by setting `--binding-address` and `--secure-port` flags.
 
+## Kubernetes Client Connection
+
+Descheduler's policy can be further configured to set kubeconfig file and
+client connection settings to use when communicating with the Kubernetes API Sever.
+
+_This option is only available in Descheduler version v0.27 and later._
+
+```yaml
+apiVersion: "descheduler/v1beta1"
+kind: "DeschedulerPolicy"
+clientConnection:
+  Kubeconfig: "" # optional
+  qps: 50        # optional
+  burst: 100     # optional
+```
```

Comment:
These fields were added to the componentconfig, not the DeschedulerPolicy. afaik that means users can't set this in yaml as you're describing here, just through flags